osi_streamingupdate.proto

```diff
@@ -82,7 +82,5 @@ message StreamingUpdate
     // Entities that will no longer be updated, because they are considered
     // obsolete by the sender.
     //
-    // \note IDs are globally unique.
-    //
```

Comment:
This should stay, as the IDs this applies to are globally unique.

Reply:
But then this prone to be misunderstood. Because the IDs this points to are unique, but because per definition these IDs already exist, this ID field, where the note is, is not unique.

Reply:
I fixed it with the suggestion of @thomassedlmayer in the conversation below.